man/systemd-sleep.conf.xml

```diff
@@ -145,12 +145,9 @@
 
       <varlistentry>
         <term><varname>HibernateMode=</varname></term>
-        <term><varname>HybridSleepMode=</varname></term>
 
-        <listitem><para>The string to be written to <filename>/sys/power/disk</filename> by, respectively,
-        <citerefentry><refentrytitle>systemd-hibernate.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>
-        and
-        <citerefentry><refentrytitle>systemd-hybrid-sleep.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>.
+        <listitem><para>The string to be written to <filename>/sys/power/disk</filename> by <citerefentry>
+        <refentrytitle>systemd-hibernate.service</refentrytitle><manvolnum>8</manvolnum></citerefentry>.
```

Comment:
Hmm, but `systemd-hibernate.service` now also reads this. So maybe do not remove line 153?

Reply:
This is also used by `suspend-then-hibernate` (see below), but we don't explicitly list that here either. I think it's clearer to document this in another paragraph.